aarch64_extension.cpp

```diff
@@ -45,11 +49,79 @@ class Disassembler {
 // called and the associated Capstone resources will be released
 static thread_local Disassembler disassembler;
 
+/*
+ * Given a Vector Arrangement Specifier, return the _element_ size to be
+ * operated on.
+ * Verify that the VAS is indeed valid before calling this function.
+ */
+static size_t vectorElementSize(enum arm64_vas vas) {
```

Comment:
Where is this used?

Reply:
Good catch, that was meant for the other branch. Removed.